src/jit/importer.cpp

```diff
@@ -4693,6 +4693,107 @@ void           Compiler::impImportAndPushBox (CORINFO_RESOLVED_TOKEN * pResolved
     impPushOnStack(op1, tiRetVal);
 }
 
+/*****************************************************************************
+*
+*  Build and import `new` of multi-dimmensional array
+*/
+
+void           Compiler::impImportNewObjArray(CORINFO_RESOLVED_TOKEN* pResolvedToken,
```

Comment:
Coding conventions: this needs to use a standard header, and I think it could use a slightly more detailled description.